grafana/scylla-cql.template.json

```diff
@@ -58,6 +58,7 @@
                             }
                         ],
                         "description": "Number of CQL INSERT commands generated by the user",
+                        "description": "scylla_cql_inserts",
```

Comment:
Do we expect to see 2 'description' entries? I assume the 1st is the one picked by the dashboard generator?